pkg/apis/configs/v1alpha1/configmappropagation_lifecycle_test.go

```diff
@@ -199,3 +200,25 @@ func TestConfigMapPropagationIsReady(t *testing.T) {
 		})
 	}
 }
+
+func TestSetCopyConfigMapStatus(t *testing.T) {
+	tests := []struct {
+		name  string
+		cmpsc *ConfigMapPropagationStatusCopyConfigMap
+		want  *ConfigMapPropagationStatusCopyConfigMap
+	}{{
+		name:  "all happy",
```

Comment:
If the empty string guards are kept, a test case should be added that verifies they work.